### PR TITLE
Add whitewater=portage_way to roads

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/mapzen/vector-datasource/issues/599

Depends on https://github.com/mapzen/TileStache/pull/140

@nvkelso, @zerebubuth could you review please?

### CODE REVIEW

**Comment by nvkelso** [comment]
:+1:

File: test/599-whitewater.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# node 2356117215
```
Comment (zerebubuth) [inline test/599-whitewater.py]: This node is in Europe, and so won't be in any North America extracts we use to test with. Suggest http://www.openstreetmap.org/node/3134398100 instead?

Comment (rmarianski) [inline test/599-whitewater.py]: Good call, updated in 210b24b.

**Comment by zerebubuth** [comment]
:+1:

**Pull request merged**